packages/admin/src/Resources/Pages/CreateRecord.php

```diff
@@ -81,7 +83,7 @@ public function create(bool $another = false): void
         $this->redirect($this->getRedirectUrl());
     }
 
-    protected function getCreatedNotificationMessage(): string
+    protected function getCreatedNotificationMessage(): ?string
```

Comment:
This is a breaking change I believe.

Reply:
This isn't a breaking change - existing overrides to this method would have more narrow (but compatible typehints) - `?string -> string`

Reply:
Yeah, got it the wrong way round. If the base class had a `-> string` and the child class implemented the `-> ?string` it would fail.

Ignore me!